### DISCUSSION TITLE
Does it require the user's consent to cookies to use the Google Model Viewer on a website?

### DISCUSSION THREAD

**Discussion by Trolli991** [google/model-viewer#4828] [Q&A]
Is data sent to third-party providers such as Google during use or does the user need to consent to any cookies for other reasons in order to be GDPR-compliant?
I don't think so, but couldn't find any more detailed information about it.

Thanks!

**Comment by elalish** [comment]
No, we're just a library - no analytics, no servers of any kind. If you happen to use our CDN `https://ajax.googleapis.com/ajax/libs/model-viewer/3.5.0/model-viewer.min.js` we'll get download stats, but that's it, and of course you can always serve it yourself instead.

**Reply by Trolli991** [reply]
Thank you, that was the answer I was looking for :)
